services/business/workloads_test.go

```diff
@@ -37,23 +37,37 @@ func TestDeploymentListHandler(t *testing.T) {
 
 	assert.Equal(3, len(workloads))
 	assert.Equal("httpbin-v1", workloads[0].Name)
+	assert.Equal(true, workloads[0].AppLabel)
+	assert.Equal(false, workloads[0].VersionLabel)
 	assert.Equal("httpbin-v2", workloads[1].Name)
+	assert.Equal(true, workloads[1].AppLabel)
+	assert.Equal(true, workloads[1].VersionLabel)
 	assert.Equal("httpbin-v3", workloads[2].Name)
+	assert.Equal(false, workloads[2].AppLabel)
+	assert.Equal(false, workloads[2].VersionLabel)
 }
 
 func fakeDeploymentList() *v1beta1.DeploymentList {
+	conf := config.Get()
```

Comment:
did you setup the config on the test ? otherwise this may fail.

Reply:
I see....maybe it's a good moment to move all mocks to the same file ^^

Reply:
Anyway...this config comes from "github.com/kiali/kiali/config"  and it's working

Reply:
There is a separated JIRA for that to be planned in Sprint 11.
But yes, it's some of the areas identified to improve.